android-runtime/app/build.gradle

```diff
@@ -0,0 +1,40 @@
+apply plugin: 'com.android.application'
```

Comment:
This app still needs to be moved somewhere else. Only the runtime itself should be in this package